DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: mlr3extralearners
 Title: Extra Learners For mlr3
-Version: 0.6.0-9000
```

Comment:
Please leave it at 0.6.0-9000, I will increment the version when I actually make the new release.